### PR TITLE
RU translations for 2.4.2

### PR DESCRIPTION
**Pull request by wduda**
Add strings for RU localization.
Some cosmetic changes in code.

### CODE REVIEW

**Review by wduda — COMMENTED** [review]
Reviewed and approved

File: src/SkillData.lua [hunk 1]
```diff
@@ -2340,10 +2504,14 @@ function TravelDictionary:CreateDictionaries()
         FR={ name="Retour à Bree",      desc="Permet de retourner",                label="Bree", zlabel="Porte Ouest",     zone="Bree", },
         RU={ name="Возвращение в Бри",  desc="Вы можете быстро перенестись в Бри", label="Бри",  zlabel="Западные ворота", zone="Бри", },
         map={MapType.ERIADOR, 525, 335},
-        acquire={ {
-            EN={deed="Enmity of the Wargs II",},
-            DE={deed="",},
-            FR={deed="",}} },
+        acquire={
+            {
```
Comment (whiterabbit963) [inline src/SkillData.lua:2508]: There is extra formatting on these Enmity of tags

File: src/LocaleStrings.lua [hunk 2]
```diff
@@ -403,7 +403,7 @@ LC_RU.menuSkills = "Проверить доступность навыков"
 LC_EN.acquired = "You have acquired the (.*) skill%."
 LC_DE.acquired = "Ihr habt Euch die Fertigkeit (.*) angeeignet%."
 LC_FR.acquired = "Vous avez acquis la compétence (.*)%."
-LC_RU.acquired = "You have acquired the (.*) skill%."
+LC_RU.acquired = "Вы приобрели навык (.*)%."
```
Comment (whiterabbit963) [inline src/LocaleStrings.lua:406]: Has this text been confirmed and/or tested in an RU client by performing a skill acquisition and observing the skill getting added to the main window?

File: src/SkillData.lua (hunk 1 above)
Comment (wduda) [reply src/SkillData.lua:2508]: thanks, fixing

File: src/LocaleStrings.lua (hunk 2 above)
Comment (wduda) [reply src/LocaleStrings.lua:406]: Have not been able to test this. @Svertalf, can you help us understand how you achieved this?

Comment (wduda) [reply src/LocaleStrings.lua:406]: @Svertalf if you are not aware, this has to be the exact string as posted in chat by the game. I tested with the RU client using the downloaded DAT files straight from discord, and it is different from what you posted.

Comment (whiterabbit963) [reply src/LocaleStrings.lua:406]: I tested PR #215 in the RU client (downloaded dat files from discord).

Comment (wduda) [reply src/LocaleStrings.lua]: FInal version is ```LC_RU.acquired = "Вы приобрели умение <(.*)%>."```. This way skill acquisition detection in RU will start working.

**Pull request merged**